WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -272,7 +274,9 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
 
     /// Returns an instance of a SupportView, configured to be displayed from a specified Support Source.
     ///
-    func presentSupport(from sourceViewController: UIViewController, sourceTag: WordPressSupportSourceTag) {
+    func presentSupport(from sourceViewController: UIViewController, sourceTag: WordPressSupportSourceTag,
```

Comment:
This one seemed to be updated in the previous version, so I just adapted the conformance to this protocol. Not sure why it wasn't already updated